src/AccessibilityInsights.SetupLibrary/DirectoryManagement.cs

```diff
@@ -11,46 +11,16 @@ namespace AccessibilityInsights.SetupLibrary
     /// </summary>
     public static class DirectoryManagement
     {
-        /// <summary>
-        /// Create necessary folders for Configurations and User data
-        /// </summary>
-        public static void CreateFolders()
-        {
-            CreateFolder(sUserDataFolderPath);
-            CreateFolder(sConfigurationFolderPath);
-        }
-
         /// <summary>
         /// Create folder if it doesn't exist. 
         /// </summary>
         /// <param name="path"></param>
-        private static void CreateFolder(string path)
+        public static void CreateFolder(string path)
```

Comment:
>CreateFolder [](start = 27, length = 12)

It feels odd to have this as the only method in this class. Could we reasonably put this method into another class?